okapi-core/src/main/java/org/folio/okapi/bean/ModuleInstance.java

```diff
@@ -86,4 +88,20 @@ public ModuleInstance withRetry() {
     this.withRetry = true;
     return this;
   }
+
+  public String getxOkapiUserId() {
+    return xokapiUserid;
+  }
+
+  public void setxOkapiUserId(String xokapiUserid) {
+    this.xokapiUserid = xokapiUserid;
```

Comment:
Can you use consistent case of id, either always upper or always lower in method, parameter and variable?

Reply:
Good catch - typo